### PR TITLE
chore(modules-sdk): Custom Module definition

### PR DESCRIPTION
**Pull request by carlos-r-l-rodrigues**

### CODE REVIEW

File: packages/core/modules-sdk/src/definitions.ts [hunk 1]
```diff
@@ -71,15 +71,15 @@ export const MODULE_PACKAGE_NAMES = {
...
+  [Modules.INVENTORY]: "@medusajs/inventory-next",
   [Modules.LINK]: "@medusajs/link-modules",
   [Modules.PAYMENT]: "@medusajs/payment",
   [Modules.PRICING]: "@medusajs/pricing",
   [Modules.PRODUCT]: "@medusajs/product",
   [Modules.PROMOTION]: "@medusajs/promotion",
   [Modules.SALES_CHANNEL]: "@medusajs/sales-channel",
   [Modules.FULFILLMENT]: "@medusajs/fulfillment",
-  [Modules.STOCK_LOCATION]: "@medusajs/stock-location-next", // TODO: To be replaced when current `@medusajs/stock-location` is deprecated
+  [Modules.STOCK_LOCATION]: "@medusajs/stock-location-next",
```
Comment (olivermrbl) [inline packages/core/modules-sdk/src/definitions.ts]: question: Do you think we will remember without them? 😄

Comment (carlos-r-l-rodrigues) [reply packages/core/modules-sdk/src/definitions.ts]: hahaha It is unlikely. I will add them back.

File: packages/medusa/src/loaders/index.ts [hunk 2]
```diff
@@ -45,45 +40,6 @@ type Options = {
   isTest: boolean
 }
 
-async function loadLegacyModulesEntities(configModules, container) {
```
Comment (sradevski) [inline packages/medusa/src/loaders/index.ts]: I already removed all of this here: https://github.com/medusajs/medusa/pull/7295 so there might be some rebase conflict resolution to do

**Review by adrien2p — APPROVED** [review]

**Review by sradevski — APPROVED** [review]
Nice to see things getting cleaned up!

**Pull request merged**